doc/installing.rst

```diff
@@ -4,182 +4,181 @@ Installation
-* `ginga <https://pypi.org/project/ginga/>`_ -- version 3.0 or later
-* `linetools <https://pypi.org/project/linetools/>`_ -- version 0.3.dev2231 or later
-* `pytest <https://pypi.org/project/pytest/>`_ -- version 3.0.7 or later
-* `shapely <https://pypi.org/project/Shapely/>`_ -- version 1.7 or later; optional, **required for KCWI only**
+* `linetools <https://pypi.org/project/linetools/>`_ -- version 0.2 or later
+* `extension_helpers <https://pypi.org/project/extension-helpers/>`_ -- version 0.1 or later
+* `qtpy <https://github.com/spyder-ide/qtpy>`_ -- version 1.9 or later
+
+The interactive tools in ``PypeIt`` are built using the `QT <https://www.qt.io/>`_ windowing toolkit. The ``qtpy`` package
+is used to provide an abstract interface to the two most widely used QT bindings for Python:
+
+* `pyqt5 <https://riverbankcomputing.com/software/pyqt/intro>`_ -- version 5
+* `PySide2 <https://wiki.qt.io/Qt_for_Python>`_ -- version 5
+
+At least one of those bindings must be installed for the interative GUIs to work. DO NOT INSTALL BOTH, as these
+two packages do not play nicely together. We strongly recommend that you go with pyqt5, unless you are attracted
+to the more flexible licensing that PySide2.  PySide2 can occasionally cause GUIs to crash because
+of conflicts with other packages in your environment that use pyqt5. 
 
 Developer-only items
 --------------------
 
-If you are developing, you may need the following packages for building the documentation:
+If you are developing, you will need the following packages:
+
+* `pytest <https://pypi.org/project/pytest/>`_ -- version 3.0.7 or later
+* `tox <https://tox.readthedocs.io/en/latest/>`_
+
+Building the documentation requires the following extra dependencies:
 
 * `sphinx <https://www.sphinx-doc.org/en/master/>`_ -- version 4.0 or later
-* sphinx-automodapi
-* sphinx_rtd_theme
+* sphinx_automodapi (pip install only)
+* sphinx_rtd_theme (pip install only)
 
-How to install dependencies
----------------------------
+Installing via ``pip`` (recommended)
+------------------------------------
 
-#. Download `requirements.txt <https://github.com/pypeit/PypeIt/blob/master/pypeit/requirements.txt>`__.
+The recommended method for installing ``PypeIt`` and its dependencies, both required and optional,
+is via `pip <https://pypi.org/project/pip/>`_. It is very highly recommended to first set up a clean environment
+in which to install ``PypeIt`` so that possible dependency conflicts can be avoided. This can be done via ``virtualenv``:
 
-#. Install the dependencies using ``conda``::
+        virtualenv pypeit
+        source pypeit/bin/activate
 
-        conda install -y -c conda-forge --file requirements.txt
+or via ``conda``:
 
-  or ``pip``::
+        conda create -n pypeit
+        conda activate pypeit
 
-        pip install -r requirements.txt
+See the `Virtualenv documentation <https://virtualenv.pypa.io/en/latest/>`_ and/or `Managing Environments with Conda
```

Comment:
Thanks for pointing this out.  This actually encouraged me to switch from `pyenv` to `virtualenv`.  Here, we might also suggest using `virtualenvwrapper`: https://virtualenvwrapper.readthedocs.io/en/latest/ .

Reply:
i use `conda` personally, but i'll check that out and update the docs.